### PR TITLE
docs: oryx versioning

### PR DESCRIPTION
**Pull request by tolerants**
Doc about Oryx versioning and release policies

closes https://spryker.atlassian.net/browse/HRZ-2147

### CODE REVIEW

**Review by tobi-or-not-tobi — CHANGES_REQUESTED** [review]

File: _drafts/oryx/versioning.md [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+
+## Release Process
+The Oryx framework comprises three distinct components, each of which is released separately:
+- Libraries
+- Application
+- Labs
+
+### Libraries
+The primary sources of functionality within the Oryx frameworks are the Oryx libraries, which include core features, utilities and helpers, domain-specific functionality, as well as pre-designed themes and presets that can be readily used. 
+All the [packages](//TODO: add link) released together under the same version.
```
Comment (tobi-or-not-tobi) [inline _drafts/oryx/versioning.md:30]: we agreed on adding future links as an html comment

>

File: _drafts/oryx/versioning.md [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+- Libraries
+- Application
+- Labs
+
+### Libraries
+The primary sources of functionality within the Oryx frameworks are the Oryx libraries, which include core features, utilities and helpers, domain-specific functionality, as well as pre-designed themes and presets that can be readily used. 
+All the [packages](//TODO: add link) released together under the same version. 
+
+### Applications
+Oryx applications serve as boilerplate templates for specific domains or areas, such as b2c storefronts, fulfillment applications, and more.
```
Comment (tobi-or-not-tobi) [inline _drafts/oryx/versioning.md:33]: This seems wrong, let's consult with @supproduction on what this is exactly.

Comment (supproduction) [reply _drafts/oryx/versioning.md:33]: Looks like all right

File: _drafts/oryx/versioning.md [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+template: concept-topic-template
+---
+
+# Versioning
+In this document, we outline the methods we use to deliver an advanced application development platform while maintaining stability. Our goal is to ensure that any upcoming changes are introduced in a predictable manner. We want all Oryx users to be informed about the addition of new features and adequately prepared for the removal of outdated ones.
+
+Stability is a critical aspect of any software framework, and it's especially important for frontend frameworks like Oryx. The stability of a framework refers to its ability to function reliably and consistently over time, despite changes in the environment or updates to its components. 
+
+Ensuring that a framework is stable requires careful planning, testing, and versioning. In this article, we will explore how the Oryx framework achieves stability through its versioning and release processes. We'll look at how Oryx manages its releases, how it uses version numbers to indicate changes, and how it ensures backward compatibility for its users. 
+
```
Comment (dunqan) [inline _drafts/oryx/versioning.md:13]: Wondering, if you should not mention about rc versions. As it's related to both mentioned releasing and stability.

Comment (tolerants) [reply _drafts/oryx/versioning.md:13]: Good point.

File: _drafts/oryx/versioning.md (hunk 2 above)
Comment (tolerants) [reply _drafts/oryx/versioning.md:37]: Contacted Alexey, but it looks like he don't really know what you're talking about. Can you specify your point @tobi-or-not-tobi ?

**Review by tobi-or-not-tobi — DISMISSED** [review]

File: _drafts/oryx/versioning.md [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+## Release Process
+The Oryx framework comprises two distinct components, each of which is released separately:
+- Libraries
+- Labs
+
+### Libraries
+The primary sources of functionality within the Oryx frameworks are the Oryx libraries, which include core features, utilities and helpers, domain-specific functionality, as well as pre-designed themes and presets that can be readily used. 
+All the [packages](https://www.npmjs.com/org/spryker-oryx) released together under the same version. 
+
+### Labs package
```
Comment (andriitserkovnyi) [inline _drafts/oryx/versioning.md:35]: Is the `labs package` an uncommon name? 

The libraries are also delivered in packages, but the previous section is called just `Librarries`, not `Labraries package`.

**Review by jmartinspryker — APPROVED** [review]

**Review by jmartinspryker — APPROVED** [review]

File: _drafts/oryx/oryx-versioning.md [hunk 5]
```diff
@@ -0,0 +1,71 @@
...
+
+### Labs
+
+The Oryx labs consist of experimental or demo functionality. Its version is tied to the current version of the libraries and is never considered stable. Therefore, the labs version number consists of three parts:
+
+```
+0.{Libraries-version}.{patch}
+```
+
+For example, if the current libraries version is `1.5.2`, the version of the labs is `0.152.0`. If we release the next version of labs before releasing the libraries, the version of the labs is updated to `0.152.1`.
```
Comment (VadymSachenko) [inline _drafts/oryx/oryx-versioning.md:46]:
```suggestion
For example, if the current libraries; version is `1.5.2`, the version of the labs is `0.152.0`. If we release the next version of labs before releasing the libraries, the version of the labs is updated to `0.152.1`.
```

File: _drafts/oryx/oryx-versioning.md [hunk 6]
```diff
@@ -0,0 +1,71 @@
...
+Oryx follows the principles of [semantic versioning](https://semver.org/).
+
+This means that, for version `x.y.z`, the following applies:
+- If critical bug fixes are released, a patch release is made by changing the `z` number. For example, `1.5.2` to `1.5.3`.
+- If new features or non-critical fixes are released, a minor release is made by changing the `y` number. For example, `1.5.2` to `15.6.0`.
+- If significant changes that may break compatibility are released, a major release is made by changing the `x` number. For example, `1.5.2` to `2.0.0`.
+
+### Release previews
+
+For those who want to know what's to come beforehand, for every major and minor release, we offer pre-release versions called Release Candidates (RC).
```
Comment (VadymSachenko) [inline _drafts/oryx/oryx-versioning.md:24]: If possible, use the closed form of compound words  - https://developers.google.com/style/word-list#closed-compounds-and-prefixes
```suggestion
For those who want to know what's to come beforehand, for every major and minor release, we offer prerelease versions called Release Candidates (RC).
```

File: _drafts/oryx/oryx-versioning.md [hunk 7]
```diff
@@ -0,0 +1,71 @@
+---
+title: Oryx versioning
+description: Oryx framework uses semantic versioning to ensure stability
+template: concept-topic-template
```
Comment (VadymSachenko) [inline _drafts/oryx/oryx-versioning.md:4]: I this doc requires the "last_updated" tag, please add it.

**Review by VadymSachenko — CHANGES_REQUESTED** [review]

**Pull request merged**